src/main/kotlin/blackjack/domain/Dealer.kt

```diff
@@ -1,16 +1,21 @@
 package blackjack.domain
 
 class Dealer : Participant("딜러") {
-    override fun isDealer(): Boolean {
-        return true
-    }
+    override val isDealer: Boolean = true
 
     override fun canNotReceiveCard(): Boolean {
         return score > MAXIMUM_SCORE_TO_RECEIVE_CARD
     }
 
     fun determineResult(gamblerResults: List<GamblerResult>): DealerResult {
-        return DealerResult(this, gamblerResults)
+        val profit = gamblerResults.sumOf { gamblerResult -> gamblerResult.profit }
+            .reverseSign()
+
+        return DealerResult(this, profit)
+    }
+
+    private fun Double.reverseSign(): Double {
+        return -this
```

Comment:
이미 kotlin 에서는 `-`를 operator fun인 `unaryMinus()`를 활용하면 불필요하게 확장함수를 정의하지 않아도 될 것 같아요. 🤔 

```suggestion
        val profit = gamblerResults.sumOf { gamblerResult -> gamblerResult.profit }
            .unaryMinus()

        return DealerResult(this, profit)
```

Reply:
오......이런게 있었군요 !! 감사합니다. 함수를 구현할 때는 기본으로 제공되는 함수가 있는지 한번 더 살펴보는것이 좋겠네요!